### DISCUSSION TITLE
Translations for 3.18.0

### DISCUSSION THREAD

**Discussion by fmasa** [fmasa/wfrp-master#310] [General]
Hi @TheoKDK, @Stilzkin, @Joly0,
I recently added Diseases to the app (among some smaller features), which brought a chunk of new texts. Do you think you will find some time to take a look at the translations?

I know you spent a lot of time on existing translations and I don't want to release new version with whole Character tab not translated without at least letting you know.

**Comment by Joly0** [comment]
Sure, will take a look and try to translate it. Will let you know once i am finished

**Comment by Stilzkin** [comment]
OK no problem, i will tell you when finish :)

**Reply by Stilzkin** [reply]
@fmasa OK ive done.

**Reply by fmasa** [reply]
@Stilzkin Thanks a lot! Merged :slightly_smiling_face:

**Comment by fmasa** [comment]
Just a heads up: I plan to release 3.18.0 on Sunday.
